src/loc/vtp/Nominate.css

```diff
@@ -0,0 +1,15 @@
+#NominateButton {
```

Comment:
Why using IDs here? This means that there will always be a single Nominate button per page (which indeed is the case currently).

Reply:
Because css rules defined on ID's take precedence on those defined on classes, and that's the only way I was able to override padding for this component:
`    padding: 4px 15px;`
Maybe there are other ways...

Reply:
OK